sassoftware/vscode-sas-extension
# feat: support notebook

**Summary**
Support #174

**Testing**

- New file > SAS Notebook
- Run code and see log and HTML output
- Support SAS, SQL and Python code
- Execution can be interrupted
- Save to `.sasnb` file and open again
